### DISCUSSION TITLE
Mayo CLI

### DISCUSSION THREAD

**Discussion by mdemirci-exar** [fougue/mayo#301] [Q&A]
I was unable to build the mayo-conv CLI version, likely because I couldn't build OpenCASCADE properly. As a result, I'm using mayo.exe in the terminal, but I can't read the output via stdout or stderr—I suspect it uses a buffer. Could you add the mayo-conv CLI version to the releases? Additionally, can the output be read via stdout or stderr with the mayo-conv CLI?

**Comment by HuguesDelorme** [comment]
Hello @mdemirci-exar
If you are using Mayo v0.8.0 then the CLI converter is provided also by `mayo.exe`(single executable for both the GUI app and the CLI converter)
Current development version(coming v0.9.0) splits the GUI app `mayo.exe` and the CLI converter `mayo-conv.exe`
So just run `mayo.exe` with the relevant command-line arguments

**Comment by mdemirci-exar** [comment]
Hello,

I cant read any console output.
```
import subprocess

cmd = [
    r"C:\Users\******\Desktop\mayo\mayo\mayo.exe",
    r"C:\Users\******\Desktop\*****.STEP",
    "-s", r"C:\Users\******\Desktop\mayo\mayo\option.ini",
    "-e", r"C:\Users\******\Desktop\*****.glb"
]

process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
stdout, stderr = process.communicate()

print("STDOUT:", stdout.decode())
print("STDERR:", stderr.decode())

```

**Reply by HuguesDelorme** [reply]
Hello have a look at this issue, maybe it can help;
https://github.com/fougue/mayo/issues/232

Mayo CLI output is not designed to be parsed by other applications, it's 100% targeted to the end-user.
This should be changed so CLI output can be formated to parsable text.
